base_layer/core/src/base_node/sync/header_sync/synchronizer.rs

```diff
@@ -284,6 +284,9 @@ impl<'a, B: BlockchainBackend + 'static> HeaderSynchronizer<'a, B> {
                     split_info.local_tip_header.height(),
                     split_info.remote_tip_height,
                     sync_peer.node_id(),
+                    client
+                        .get_last_request_latency()
```

Comment:
As discussed, `get_last_request_latency` only measures the last RTT which is not available during streaming, to measure time between incoming header/blocks/etc you'll need to use an `Instant` and `Instant::elapsed` to measure that yourself.

Reply:
We can change this after you latency PR to use the same metric.